auth_backends/strategies.py

```diff
@@ -0,0 +1,17 @@
+from django.conf import settings
+
+from social.strategies.django_strategy import DjangoStrategy
+
+
+class CurrentSiteDjangoStrategy(DjangoStrategy):
```

Comment:
You'll want to look at python-social-auth for ideas regarding testing. The method is pretty straightforward, so no need to go crazy beyond just making sure we handle existing settings and properly fail when a setting doesn't exist.